src/compiler/AST/index.js

```diff
@@ -393,7 +393,7 @@ export function memory(limits: Limit, id: ?Index): Memory {
 
 export function data(
   memoryIndex: Index,
-  offset: Array<Instruction>,
+  offset: Instruction,
```

Comment:
see #147 - data segments also only support a single instruction

Reply:
I didn't it for WAST where the definition is: `data:    ( data <var>? ( offset <instr>* ) <string>* )`.

I'm using some transform (https://webassembly.js.org/docs/contrib-transform-wast-identifier-to-index.html) to handle these differences.